cmd/juju/dashboard/dashboard.go

```diff
@@ -113,21 +100,8 @@ func (c *dashboardCommand) Run(ctx *cmd.Context) error {
 		return errors.Trace(err)
```

Comment:
Do we still need this command?
This dashboardURL does not work anymore.

Reply:
Yeah, it will be modified once the dashboard charm a bit further along. It will behave similarly as for k8s - it will block and proxy to the dashboard endpoint.

Reply:
Added a TODO to the command